src/test/java/collectionsEducation/CountryWorker.java

```diff
@@ -4,88 +4,189 @@
 
 public class CountryWorker {
 
+
     public static void main(String[] args) {
         List<String> names = TestDataBuilder.getCountriesNamesList();
         List<CountryDTO> allCountries = TestDataBuilder.getCountriesList();
 //        тут будеш визивать потрібний метот
+        System.out.print("LIST COUNTRY :");
+
+        for (String test : names) {
+                System.out.print(test + " ,");
+        }
+        System.out.println('\n');
+        System.out.println("Задача1 :" + getListOfNamesEquals(names,"United Arab Emirates"));
+
         System.out.println();
-    }
+        System.out.println("Задача2 :" + getNameEquals(names,"GerManY"));
+
+        System.out.println();
+        System.out.println("Задача3 :" + getSortListOfNames(names,false));
+
+        System.out.println();
+        System.out.println("Задача4 :" + getListOfCountriesNameEquals(allCountries,"iTaLy"));
+
+        System.out.println();
+        System.out.println("Задача5 :" + getListOfCountriesNameContains(allCountries,"uniTed"));
+
+        System.out.println();
+        System.out.println("Задача6 :" + getListOfCountriesByName(allCountries,"united",false));
+
+        System.out.println();
+        System.out.println("Задача7 :" + getListOfCountriesByLanguage(allCountries,"fra"));
 
-    private List<String> getListOfNamesEquals(List<String> names, String name) {
+        System.out.println();
+        System.out.println("Задача8 :" + getListOfCountriesByLanguageMap(allCountries,"eng"));
+
+        System.out.println();
+        System.out.println("Задача9 :" + getListOfCountriesByLanguageMap(allCountries));
+
+
+
+
+
+    }
+    private static List<String> getListOfNamesEquals(List<String> names, String name) {
         List<String> filteredCountries = new ArrayList<>();
-//    Замість мого коментаря повинна бути реалізація
+
+        for (String country : names) {
+            if(country.equalsIgnoreCase(name)){
+                filteredCountries.add(country);
+            }
+        }
+        //    Замість мого коментаря повинна бути реалізація
 //    Задача 1: знайди всі імена, які містить введене значення, наприклад united.
 //    Пошук повинен бути case insensitive - тобто буде працювати однаково для united, UNITED і United
+
         return filteredCountries;
     }
 
-    private String getNameEquals(List<String> names, String name) {
+    private static String getNameEquals(List<String> names, String name) {
```

Comment:
Solution is incorrect, your solution will return last found country and you need first. 

If your list will be 
 ```
public static List<CountryDTO> getCountriesList() {
        List<CountryDTO> countriesList = new ArrayList<>();
        countriesList.add(new CountryDTO().setName("United States of America").setLanguage("ENG"));
        countriesList.add(new CountryDTO().setName("United Kingdom").setLanguage("ENG"));
        countriesList.add(new CountryDTO().setName("United Arab Emirates").setLanguage("ARA"));
        countriesList.add(new CountryDTO().setName("Mexico").setLanguage("SPA"));
        countriesList.add(new CountryDTO().setName("Spain").setLanguage("SPA"));
        countriesList.add(new CountryDTO().setName("Germany").setLanguage("GER"));
        countriesList.add(new CountryDTO().setName("Italy").setLanguage("ITA"));
        countriesList.add(new CountryDTO().setName("Canada").setLanguage("FRA"));
        countriesList.add(new CountryDTO().setName("Canada").setLanguage("ENG"));
        countriesList.add(new CountryDTO().setName("French").setLanguage("FRA"));
        return countriesList;
    }
```

Your method will return countriesList.add(new CountryDTO().setName("Canada").setLanguage("ENG"));, but the task is return countriesList.add(new CountryDTO().setName("Canada").setLanguage("FRA"));

You should add additional return statement